src/v1/json/salutes.json

```
@@ -1,24 +1,56 @@
 {
   "salutes": [
+    "Stone.",
     "Rock on!",
+    "We rock!",
     "For Karl!",
+    "Rock solid!",
+    "For Teamwork!",
+    "By the Beard!",
+    "Rock and roll!",
+    "Rock... Solid!",
     "Rock and Stone!",
+    "Rock and Stone.",
+    "We\u0027re the best!",
```

Remove the unnecessary Unicode
```suggestion
    "We're the best!",
```